modin/pandas/base.py

```diff
@@ -4503,27 +4503,32 @@ def set_backend(
         self_backend = self.get_backend()
         normalized_backend = Backend.normalize(backend)
         if normalized_backend != self_backend:
+            max_rows, max_cols = self._query_compiler._max_shape()
+            if switch_operation is None:
+                desc = (
+                    f"Transferring data from {self_backend} to {normalized_backend}"
+                    + f" with max estimated shape {max_rows}x{max_cols}"
+                )
+            else:
+                desc = (
+                    f"Transferring data from {self_backend} to {normalized_backend} for"
+                    + f" '{switch_operation}' with max estimated shape {max_rows}x{max_cols}"
+                )
+
             try:
                 from tqdm.auto import trange
 
-                max_rows, max_cols = self._query_compiler._max_shape()
-                if switch_operation is None:
-                    desc = (
-                        f"Transferring data from {self_backend} to {normalized_backend}"
-                        + f" with max estimated shape {max_rows}x{max_cols}"
-                    )
-                else:
-                    desc = (
-                        f"Transferring data from {self_backend} to {normalized_backend} for"
-                        + f" '{switch_operation}' with max estimated shape {max_rows}x{max_cols}"
-                    )
                 progress_iter = iter(trange(progress_split_count, desc=desc))
             except ImportError:
-                # Iterate over blank range(2) if tqdm is not installed
-                pass
+                # Fallback to simple print statement when tqdm is not available
+                print(desc)  # noqa: T201
```

Comment:
It seems like tqdm's bar puts its text in stderr (a file produced by a `>` redirect did not show it, but one with `2>` did). IMO we should mimic this behavior.

```suggestion
                print(desc, file=sys.stderr)  # noqa: T201
```

We should also make an option to allow switch progress to occur silently. Users have complained about over-logging before, and I imagine non-interactive users would appreciate an option to keep stderr and stdout clean.

Reply:
> IMO we should mimic this behavior.

Done.

> We should also make an option to allow switch progress to occur silently. 

Done.